src/firebaseApi.ts

```diff
@@ -0,0 +1,51 @@
+import * as api from "./api";
+
+const API_VERSION = "v1beta1";
+
+/**
+ * Represents the FirebaseProject resource returned from calling
+ * `projects.get` in Firebase Management API:
+ * https://firebase.google.com/docs/projects/api/reference/rest/v1beta1/projects#FirebaseProject
+ */
+export interface FirebaseProject {
+  projectId: string;
+  projectNumber: number;
+  displayName: string;
+  name: string;
+  resources: {
```

Comment:
I'd treat none of the things in here as guaranteed, so `hostingSite?: string` etc.

`locationId` can definitely be null atm (that's deferred location, right?) but I think other stuff will eventually be that way too.

Reply:
Ah, good point. Changed to mark all these optional.